Source/Scripts/cheat_core_items.lua

```diff
@@ -414,6 +414,67 @@ function cheat:cheat_damage_all_items(line)
   return false
 end
 
+-- Used by cheat_save_all_items and cheat_load_all_items
+_global_items_size = 1
+_global_items_classes = {}
+_global_items_amounts = {}
```

Comment:
Please put these variables into the cheat namespace. i.e. cheat.g_items_size. The mod avoids altering the lua global namespaces so we avoid conflicts with the game's scripts or other mods. There are examples of global vars here https://github.com/pryans/kcd-cheat/blob/master/Source/Scripts/cheat_util.lua#L91 and here https://github.com/pryans/kcd-cheat/blob/master/Source/Scripts/cheat_util.lua#L167, I'm assuming g_ is the prefix for the globals within the cheat namespace. I can't really remember.